suite-native/app/app.config.ts

```diff
@@ -99,6 +101,7 @@ export default ({ config }: ConfigContext): ExpoConfig => {
                         },
                     },
                 },
+                UIRequiredDeviceCapabilities: ['armv7'],
```

Comment:
Why this change?

Reply:
This was missing from expo migration and was causing error in submission to the app store connect. Basically app store's complaint was that we're changing device compatibility. https://expo.dev/accounts/trezorcompany/projects/trezor-suite/submissions/1e284324-933c-4f15-83df-373e6d8808dd
<img width="1041" alt="image" src="https://github.com/trezor/trezor-suite/assets/36101761/997b3827-d751-43ac-9015-6d1a250998a7">